vue/components/organisms/pickers/pickers.vue

```diff
@@ -38,7 +38,18 @@
                         {{ localeModel(part, "no_" + part) }}
                     </p>
                 </li>
-                <slot />
+                <li
+                    class="part button button-part button-extra"
+                    v-for="partButton in afterPartsButtons"
+                    v-bind:key="partButton.id"
+                    v-on:click="onAfterPartsButtonClick(partButton.event)"
+                >
+                    <slot v-bind:name="`part-button-${partButton.id}`">

```

Comment:
I know you probably called this `part-button` because that's what I wrote in some comment, but if the class is `button-part` then this (and the prop) must be too.

Also, we probably should be able to add this both before and after the parts.